From 85da17716d1532fd95598d8c3d6a8687897f6d58 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B8ren=20Sandmann=20Pedersen?= <ssp@redhat.com>
Date: Fri, 2 Aug 2013 09:58:58 -0400
Subject: [PATCH] compat driver: Make sure to initialize the VGA functions

---
 src/compat/compat-qxl_driver.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/compat/compat-qxl_driver.c b/src/compat/compat-qxl_driver.c
index 0047382..5b51ce4 100644
--- a/src/compat/compat-qxl_driver.c
+++ b/src/compat/compat-qxl_driver.c
@@ -1387,6 +1387,7 @@ compat_qxl_pre_init(ScrnInfoPtr pScrn, int flags)
     /* VGA hardware initialisation */
     if (!vgaHWGetHWRec(pScrn))
         return FALSE;
+    vgaHWSetStdFuncs (VGAHWPTR (pScrn));
 
     /* hate */
     compat_qxl_unmap_memory(compat_qxl, scrnIndex);
-- 
1.7.1

